packages/dx-react-chart-demos/src/demo-sources/export/material-ui/demo.jsx

```diff
@@ -0,0 +1,297 @@
+        <IconButton
+          id="iconButton"
+          onClick={handleClick}
+          className={classes.button}
+        >
+          <MoreVertIcon />
+        </IconButton>
+        <Menu
+          anchorEl={anchorEl}
+          open={open}
+          onClose={handleClose}
+          PaperProps={{
+            style: {
+              maxHeight: ITEM_HEIGHT * 4.5,
+              width: 150,
+            },
+          }}
+        >
+          {options.map(option => (
+            <MenuItem key={option} onClick={handleExport(option)}>
+              {option === 'Print' ? option : `Save as ${option}`}
+            </MenuItem>
+          ))}
+        </Menu>
+      </Template>
+    </Plugin>
+  );
+};
+
+const Export = withStyles(styles)(ExportBase);
+
+const BarWithLabel = withStyles({
+  label: {
+    fill: '#ffffff',
+    fontSize: '10px',
+    animation: `${getLabelAnimationName()} 1s`,
+  },
+})(({ classes, value, ...restProps }) => (
+  <React.Fragment>
+    <BarSeries.Point {...restProps} />
+    <Chart.Label
+      x={restProps.x}
+      y={(restProps.y + restProps.y1) / 2}
+      dominantBaseline="middle"
+      textAnchor="middle"
+      className={classes.label}
+    >
+      {`${value}%`}
+    </Chart.Label>
+  </React.Fragment>
+));
+const LegendRoot = withStyles({
+  root: {
+    display: 'flex',
+    margin: 'auto',
+    flexDirection: 'row',
+  },
+})(({ classes, ...restProps }) => (
+  <Legend.Root {...restProps} className={classes.root} />
+));
+
+const LegendItemRoot = withStyles({
+  item: {
+    flexDirection: 'column',
+    marginLeft: '-2px',
+    marginRight: '-2px',
+  },
+})(({ classes, ...restProps }) => (
+  <Legend.Item {...restProps} className={classes.item} />
+));
+const LegendLabelRoot = withStyles({
+  label: {
+    whiteSpace: 'nowrap',
+  },
+})(({ classes, ...restProps }) => (
+  <Legend.Label {...restProps} className={classes.label} />
+));
+const Marker = (props) => {
+  const { className, color } = props;
+  return (
+    <svg className={className} fill={color} width="16" height="16">
+      <circle cx={8} cy={8} r={8} />
+    </svg>
+  );
+};
+const TitleText = withStyles({
+  title: {
+    textAlign: 'center',
+    width: '100%',
+    marginBottom: '10px',
+  },
+})((props) => {
+  const { text, classes } = props;
+  const [mainText, subText] = text.split('\n');
+  return (
+    <div className={classes.title}>
+      <Typography component="h3" variant="h5">
+        {mainText}
+      </Typography>
+      <Typography variant="subtitle1">{subText}</Typography>
+    </div>
+  );
+});
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data,
+    };
+  }
+
+  render() {
+    const { data: chartData } = this.state;
+
+    return (
+      <Paper>
+        <Chart
+          data={chartData}
+        >
+          <Title
+            text={`Estimated global gaming software ${'\n'}(revenue by platform, in 2018)`}
```

Comment:
Try `\\n` instead of `${'\n'}`.

Reply:
I will change `${'\n'}` on `\n`. It will be in bootstrap and material.